package.json

```diff
@@ -4,7 +4,13 @@
   "description": "An application to power metrics for webcompat.com",
   "private": true,
   "scripts": {
-    "start": "DEBUG=webcompat-metrics:server node ./bin/www"
+    "build:client": "npm run clean && cd client; NODE_ENV=production ./node_modules/.bin/webpack --progress --colors",
+    "clean": "rm -rf client/public && mkdir client/public",
```

Comment:
Might be good to use `shx` here: https://www.npmjs.com/package/shx:

`shx rm -rf client/public && shx mkdir client/public`

That way it will work on windows still.

Reply:
good catch 👍 I’ll.

I’ll update the way to pass the env with this https://webpack.js.org/guides/environment-variables/ to work on windows.